tests/sqlite3/test-sqlite3.cpp

```diff
@@ -664,6 +665,11 @@ class test_context : public test_context_base
         return true;
     }
 
+    bool has_full_uint64_support() const SOCI_OVERRIDE
+    {
+        return false;
```

Comment:
This is rather unexpected, what's wrong with SQLite uint64 support?

Reply:
SQLite doesn't have native support for uint64. That's why you e.g. only have a `sqlite_bind_int64()` but no `sqlite_bind_uint64()` (https://www.sqlite.org/capi3ref.html).
To store a uint64 you'd have to use some kind of workaround and insert the value in a string column or similar.

Reply:
I have to correct myself there: you can store a uint64, but not the full range of this type (only up to INT64_MAX). Returning false here just disables tests where a value bigger than INT64_MAX is worked with.